api/lang/customResponses.go

```diff
@@ -26,7 +58,17 @@ func getScenarios(req *jsonrpc2.Request) (interface{}, error) {
 		return nil, err
 	}
 	file := util.ConvertURItoFilePath(params.TextDocument.URI)
-	spec, parseResult := new(parser.SpecParser).Parse(getContent(params.TextDocument.URI), gauge.NewConceptDictionary(), file)
+	content:=""
+	if isOpen(params.TextDocument.URI) {
```

Comment:
scenarios can also be fetched from infoProvider.
We don't have to parse a file to do this.